src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -104,6 +104,14 @@ function OptionRowLHN(props) {
     const shouldShowGreenDotIndicator =
         !hasBrickError && (optionItem.isUnreadWithMention || optionItem.isWaitingForTaskCompleteFromAssignee || ReportUtils.isWaitingForIOUActionFromCurrentUser(optionItem));
 
+    const sortedDisplayNames =
+        optionItem.type === CONST.REPORT.TYPE.CHAT && !optionItem.isArchivedRoom && optionItem.displayNamesWithTooltips && optionItem.displayNamesWithTooltips.length > 1
+            ? _.filter(
+                  _.map(optionItem.displayNamesWithTooltips, ({displayName}) => displayName),
+                  (displayName) => !_.isEmpty(displayName),
+              ).join(', ')
+            : '';
```

Comment:
In the LHN for small screen devices, for group chats, the names are not sorted by default in ascending order. The component used to display these names is `DisplayNamesWithoutTooltips` which makes use of the value specified for the `fullTitle` prop.

The conditions checked are for the following reasons:
1. `optionItem.type === CONST.REPORT.TYPE.CHAT` to ensure that it is actually a group chat, not an IOU or task which uses a different string value for the title. For instance, IOU titles look like `x owes y $5.00`, which is different from a list of names in a group chat.
2. `!optionItem.isArchivedRoom` to make sure that the room is not archived. Archived rooms use a different title format (eg. `Report (archived)`). 
3. The remaining two conditions are to check to make sure that there is more than 1 participant in the report, which most likely indicates that it is a group chat.

The `_.filter` was added to make sure that there are no empty string display names. I added this due to one of the tests failing.

To summarise, if the report is a group chat, and not an IOU or task, and not archived, and there is more than one participant, then the list of sorted display names will be displayed as the title for devices where the `DisplayNameWithoutTooltip` component is rendered (usually mobile / touchscreen or small screen devices).

Reply:
> Also, this name `sortedDisplayNames` seems to be confusing. `optionItem.displayNamesWithTooltips` are already sorted and you're just joining the array here. I think something like below makes more sense.
> 
> ```
> const fullTitle = condition ? join the display names : optionItem.text;
> ```
> 
> Doing this will also prevent the need for duplicate conditional when specifying `fullTittle` prop.

This won't work. `displayNamesWithTooltips` is a list of objects, which is the reason for the map. Also, one of the tests was failing with empty display names, which is why I added the filter.

Reply:
Oh, right, I see it now. Assigning to a `fullTitle` value instead of using `sortedDisplayNames`.